src/strings/en-us.json

```diff
@@ -10,6 +10,7 @@
     "AddToPlayQueue": "Add to play queue",
     "AgeValue": "({0} years old)",
     "AirDate": "Air date",
+    "AirPlay": "AirPlay",
```

Comment:
This one is actually a fix, otherwise there’s always an Airplay warning in F12 console when playing video.